labelme/widgets/canvas.py

```diff
@@ -476,7 +478,7 @@ def selectShapePoint(self, point, multiple_selection_mode):
         else:
             for shape in reversed(self.shapes):
                 if self.isVisible(shape) and shape.containsPoint(point):
-                    self.calculateOffsets(shape, point)
+                    self.calculateOffsets(self.selectedShapes, point)
```

Comment:
This can be out of the for-loop?

Reply:
Yes of course.

Reply:
Ok. Why it has to be in the for-loop?

Reply:
Why is it iterating over all `self.selectedShapes` or why it is iterating over `reversed(self.shapes)`?

1. check all shapes in `self.selectedShapes` and predict new offsets that accord all the shapes.
2. it's run only once because `480: if self.isVisible(shape) and shape.containsPoint(point)` and `490: return`